RetailCoder.VBE/App.cs

```diff
@@ -135,6 +135,11 @@ private void UpdateLoggingLevel()
 
         public void Startup()
         {
+            if (_vbe.VBProjects.Count != 0)
```

Comment:
I'd rather start parsing after we're fully initialized. Here the menus aren't even created yet, and the configuration isn't loaded; I know it's not the case right now, but we might eventually introduce parsing options (e.g. "run inspections automatically" and whatnot), ...really doing this work before we're initialized doesn't seem like the best plan.